### ISSUE TITLE
feature request: render font center instead of flush right

### ISSUE THREAD

**Issue by MichelleWhy** [sbender9/WilhelmSKSupport#9]
Right now the digital and normal font is rendered flash right inside the gauges. That sometimes  looks inconsistent, not very nice and becomes hard to read, especially when far away from screen and two gauges are close together with each gauges showing a 3 digit number.

**Comment by sbender9** [comment]
Did you try setting the “Edge Padding” in the advanced gauge options?

**Comment by MichelleWhy** [comment]
Yes. It works for single digits but not for 2 or more digits. There are gauges like Generic Ratio, who are probably set by default to show digits center. The majority of gauges do not.

**Comment by sbender9** [comment]
Can you get a screenshot?

**Comment by MichelleWhy** [comment]
1st screenshot: for STW gauge: number padding 3, edge padding 0
2nd screenshot for STW gauge: number padding 0, edge padding 50

The close it gets to center, the smaller the digits become.

![IMG_0043](https://user-images.githubusercontent.com/48654848/59112377-2fd7d980-8943-11e9-8fbc-4b0fcedf6c0c.png)

![IMG_0044](https://user-images.githubusercontent.com/48654848/59112392-36665100-8943-11e9-9d81-15ef593fb89a.png)
